packages/core/admin/admin/src/pages/MarketplacePage/utils/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+const fetchPlugins = async notify => {
+  const { data } = await axios.get(`${process.env.STRAPI_ADMIN_MARKETPLACE_API_URL}/plugins`);
```

Comment:
This will break when people run the app and don't have this variable defined. Maybe we can set the production URL as the default value? I'm not sure where is the ideal place to do it though. I couldn't find where the default value for STRAPI_ADMIN_BACKEND_URL is

Reply:
Good point I can set a default value for it in the env.js file I think